mne/viz/evoked.py

```diff
@@ -875,3 +880,158 @@ def plot_snr_estimate(evoked, inv, show=True):
+    from matplotlib.lines import Line2D
+    transFigure = fig.transFigure.inverted()
+    tf = fig.transFigure
+
+    (xt, yt) = transFigure.transform(targetax.transAxes.transform([.5, 0]))
+    (xs, _) = transFigure.transform(sourceax.transData.transform([x, 0]))
+    (_, ys) = transFigure.transform(sourceax.transAxes.transform([0, 1]))
+    return Line2D((xt, xs), (yt, ys), transform=tf, color='grey',
+                  linestyle='-', linewidth=1.5, alpha=.66, zorder=-1)
+
+
+def joint_plot(evoked, times="peaks", title='', picks=None, exclude=list(),
+               show=True, ts_args=dict(), topomap_args=dict()):
+    """Plot evoked data as butterfly plot and add topomaps for selected
+    time points.
+
+    Parameters
+    ----------
+    evoked : instance of Evoked
+        The evoked instance.
+    times : float | array of floats | "auto" | "peaks".
+        The time point(s) to plot. If "auto", 5 evently spaced topographies
+        between the first and last time instant will be shown. If "peaks",
+        finds time points automatically by checking for 3 local maxima in
+        Global Field Power.
+    title : str
+        The title.
+    picks : array-like of int | None
+        The indices of channels to plot. If None show all.
+    exclude : list of str | 'bads'
+        Channels names to exclude from being shown. If 'bads', the
+        bad channels are excluded.
+    show : bool
+        Show figure if True.
+    ts_args : dict
+        A dict of `kwargs` that are forwarded to `evoked.plot` to
+        style the butterfly plot. `axes` and `show` are ignored.
+    topomap_args : dict
+        A dict of `kwargs` that are forwarded to `evoked.plot_topomap`
+        to style the topomaps. `axes` and `show` are ignored. If `times`
+        is not in this dict, automatic peak detection is used.
+
+    Returns
+    -------
+    fig : instance of matplotlib.figure.Figure | list
+        The figure object containing the plot. If `evoked` has multiple
+        channel types, a list of figures, one for each channel type, is
+        returned.
+
+    Notes
+    -----
+    .. versionadded:: 0.12.0
+    """
+    import matplotlib.pyplot as plt
+
+    # channel selection
+    # simply create a new evoked object(s) with the desired channel selection
+    if exclude == "bads":
+        exclude = [ch for ch in evoked.info['bads']
+                   if ch in evoked.info['ch_names']]
+    evoked = evoked.copy().drop_channels(exclude)
+    if picks is not None:
+        pick_names = [evoked.info["ch_names"][pick] for pick in picks]
+        evoked = evoked.pick_channels(pick_names)
+    info = evoked.info
+    data_types = ['eeg', 'grad', 'mag', 'seeg']
+    ch_types = set([channel_type(info, idx)
+                    for idx in range(info['nchan'])
+                    if channel_type(info, idx) in data_types])
+    # if multiple sensor types: one plot per channel type, recursive call
+    if len(ch_types) > 1:
+        print(ch_types)
```

Comment:
I actually do this all the time, too :) I catch myself at least some of those times by reading through my PR diff myself one more time before asking for MRG reviews, but I don't even catch all of them that way